tests/unit/core/config/test_user_config.py

```diff
@@ -308,7 +308,7 @@ def test_sca_config_invalid_ghsa(self, local_config_path, capsys):
                     "ignored_vulnerabilities": [
                         {
                             # Invalid id
-                            "ghsa_id": "ABCD-bbbb",
+                            "identifier": "ABCD-bbbb",
```

Comment:
nit: you can maybe also change the name of the test and the docstring (since nothing indicates it's a ghsa_id).